src/client/Microsoft.Identity.Client/Internal/Logger/NullLogger.cs

```diff
@@ -15,6 +15,14 @@ internal class NullLogger : ICoreLogger
         public Guid CorrelationId { get; } = Guid.Empty;
         public bool PiiLoggingEnabled { get; } = false;
 
+        public void HealthMetric(string messageScrubbed)
```

Comment:
nit: unused params can be just [discards](https://docs.microsoft.com/en-US/dotnet/csharp/fundamentals/functional/discards).